### ISSUE TITLE
Redirect from verified email showing Error UI component

### ISSUE THREAD

**Issue by dewaele-benoit** [AxaFrance/oidc-client#1150]
### Issue and Steps to Reproduce

I used Keycloak for SSO, when user click on the redirect link from email to verify the email after create his account, the app show Error UI. 
The library does not seem to initialize the session on this process and try to parse undefined in getLoginParams function. 

I think it's the same behavior of this issue https://github.com/AxaFrance/oidc-client/issues/988, but in my opinion the solution it's to get around the problem because I need to overload the callback route or Authenticate Error Component. 

Is it possible to manage this behavior in the library or I need to try the solution in the mentioned issue ?

Step to reproduce:
1. Use Keycloak with basic settings, just require user to verify their email
2. Configure your app with keycloak 
```json
{
    "client_id": "client",
    "scope": "openid profile email offline_access",
    "redirect_uri": "http://localhost:3000/authentication/callback",
    "silent_redirect_uri": "http://localhost:3000/authentication/silent-callback",
    "authority": "https://url_to_keycloak/auth/realms/dep"
}
```  
4. Click on login button
5. Create an account in keycloak 
6. When receive the email click on the link 
7. Then UI Error appear but the user is authenticated

### Versions

"react": "^18.2.0",
"@axa-fr/react-oidc": "^7.6.0"

### Screenshots

The email :
![image](https://github.com/AxaFrance/oidc-client/assets/22936624/3e76760d-6b93-41a0-a3c9-4e90f0498b5d)

The Error UI after redirect: 
![image](https://github.com/AxaFrance/oidc-client/assets/22936624/3da672af-04c7-424f-b51d-0696136f8233)

#### Expected

To be redirected to my app

#### Actual

Show UI Error

**Comment by guillaume-chervet** [comment]
Hi @dewaele-benoit , very sorry for the delay.  

Thank you for your issue.

Do you have an example of the callback url? To you application

**Comment by abhilashlr7** [comment]
I'm facing this issue as well. In our case we use the magiclink flow. So we generate a magiclink by entering an email address, and when the email is received, we open the link in incognito mode and then the `JSON.parse(dataString)` under `initSession.ts` file. I'm guessing this will be the same issue using a service worker.

**Comment by guillaume-chervet** [comment]
Do you have a camlback link sample @abhilashlr7 ?

**Comment by abhilashlr7** [comment]
The callback component is a simple `<Loader />` and nothing fancy in it @guillaume-chervet

**Comment by guillaume-chervet** [comment]
Hi @dewaele-benoit @abhilashlr7 
Does you configuration works with this repo demo?

**Comment by abhilashlr7** [comment]
> Hi @dewaele-benoit @abhilashlr7 Does you configuration works with this repo demo?

You mean OIDC configuration?

**Comment by guillaume-chervet** [comment]
Yes @abhilashlr7 ,  I will try to write a better documentation about how to retrieve token properly. I am in holiday for a few weeks.
